semgrep_output_v1.atd

```diff
@@ -1745,6 +1745,23 @@ type ci_scan_results = {
    (* this data was originally sent to /complete, but we want to start sending
     * it to /results *)
    ?dependencies: ci_scan_dependencies option;
+
+   (* filled in by the backend to associate contributions with their corresponding
+    * source repository and driving scan*)
```

Comment:
uber-nit but there will probably be other consumers of this data, so maybe we just stick to the what instead of the why (or change the wording to "for instance ...")